coalib/settings/ConfigurationGathering.py

```diff
@@ -283,6 +288,13 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    for section_name in exclude_sections:
+        if section_name in sections:
+            del sections[section_name]
```

Comment:
Actually I'm not sure this is the right approach. Because of our coafile setting-inheritance system, this might make problems:
```
[stuff]
# settings...

[stuff.a]
# additional settings...

[stuff.a.b]
# additional settings
```
An invocation with `coala stuff.a.b !stuff.b` might mess up things, as the settings from `stuff.a` are definitely intended to exist in `stuff.a.b`.

Reply:
So how do I avoid this?

Reply:
We just want to disable execution of that section, but the inheritance chain shall not be interrupted. So I guess this is the wrong location to influence this, we directly have to modify code where sections are actually executed. Probably there will be some kind of
```python
for target in targets:
    execute_target(target)
```
I think we should return `excluded_targets` too and pass it where necessary, and then we modify the basic behaviour to
```python
for target in targets:
    if target not in excluded_targets:
        execute_target(target)
```

What is important now is that you write a test for my example (or similar) to see if coala doesn't break :+1:

Reply:
I think an Exclude Section Filter might be the answer. That way inheritance chain won't be affected and sections will just be skipped.